transport-native-io_uring/src/main/java/io/netty/channel/uring/Native.java

```diff
@@ -92,15 +91,43 @@ public void run() {
     }
 
     static RingBuffer createRingBuffer(int ringSize, Runnable submissionCallback) {
-        //Todo throw Exception if it's null
-        return ioUringSetup(ringSize, submissionCallback);
+        long[][] values = ioUringSetup(ringSize);
+        assert values.length == 2;
+        long[] submissionQueueArgs = values[0];
+        assert submissionQueueArgs.length == 11;
+        IOUringSubmissionQueue submissionQueue = new IOUringSubmissionQueue(
+                submissionQueueArgs[0],
```

Comment:
Idea is nice: I would just put somewhere the expected (constant) named indexes of the parameters.
If exposed that would help users a bit (although won't rely on the order/value of them)

Reply:
sure I can do...  That said its all "internal" so not sure if it really matters and is worth the effort.

Reply:
Matter of taste, indeed: the simplification is welcome anyway :) well done

Reply:
lets just keep it like this for now... If needed we can it later